coderd/prometheusmetrics/testdata/license-metrics.json

```diff
@@ -0,0 +1,4 @@
+{
+  "entitled=true:coderd_license_active_users": 4,
+  "entitled=true:coderd_license_user_limit": 7
```

Comment:
`coderd_license_users_{active,limit}`, while not strictly adhering to the [Prometheus naming conventions](https://prometheus.io/docs/practices/naming/), would be more auto-complete friendly.

Maybe we just have one metric `coderd_license_users {active,total}` instead?

Reply:
I like this idea, thanks for suggesting it.

Would it be possible to perform some metric math later? _Let's say: alert when active / limit = 80%._

Reply:
> As a rule of thumb, either the sum() or the avg() over all dimensions of a given metric should be meaningful (though not necessarily useful). If it is not meaningful, split the data up into multiple metrics. For example, having the capacity of various queues in one metric is good, while _mixing the capacity of a queue with the current number of elements in the queue is not._

(emphasis mine) from https://prometheus.io/docs/practices/naming/

Active users and the user limit should be different metrics, not labels on a single metric.

Reply:
They are different metrics now so resolving.